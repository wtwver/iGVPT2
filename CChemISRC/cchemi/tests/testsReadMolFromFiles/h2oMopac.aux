 START OF MOPAC FILE
 ####################################
 #                                  #
 #       Start of Input data        #
 #                                  #
 ####################################
 MOPAC_VERSION=MOPAC2012.15.146L
 DATE="Tue Dec 15 14:34:33 2015"
 METHOD=PM7
 TITLE=""
 KEYWORDS=" PM7 FORCE CHARGE=0 SINGLET BONDS AUX"
 COMMENTS="Mopac file"
 ATOM_EL[0003]=
  O  H  H 
 ATOM_CORE[0003]=
  6  1  1 
 ATOM_X:ANGSTROMS[0009]=
    0.0000    0.2956    0.0000
    0.0000    0.8745    0.7598
    0.0000    0.8746   -0.7599
 AO_ATOMINDEX[0006]=
    1    1    1    1    2    3
 ATOM_SYMTYPE[0006]=
  S PX PY PZ  S  S 
 AO_ZETA[0006]=
  5.9723  2.3490  2.3490  2.3490  1.2602  1.2602
 ATOM_PQN[0006]=
 2 2 2 2 1 1
 NUM_ELECTRONS=0008
 EMPIRICAL_FORMULA="H2 O  =     3 atoms"
 ####################################
 #                                  #
 #      Geometry optimization       #
 #                                  #
 ####################################
 ####################################
 #                                  #
 #        Final SCF results         #
 #                                  #
 ####################################
 HEAT_OF_FORMATION:KCAL/MOL=-0.577997D+02
 ENERGY_ELECTRONIC:EV=-0.469192D+03
 ENERGY_NUCLEAR:EV=+0.146513D+03
 GRADIENT_NORM:KCAL/MOL/ANGSTROM=+0.162891D+00
 POINT_GROUP=C2v 
 DIPOLE:DEBYE=+0.212913D+01
 DIP_VEC:DEBYE[3]= +0.12901D+01 +0.16938D+01 +0.54672D-17
 IONIZATION_POTENTIAL:EV=+0.121156D+02
 SPIN_COMPONENT=+0.000000D+00
 TOTAL_SPIN=+0.000000D+00
 NUMBER_SCF_CYCLES=2
 NUM_ALPHA_ELECTRONS=0
 NUM_BETA_ELECTRONS=0
 CPU_TIME:SEC=+0.781250D-02
 MOLECULAR_WEIGHT:AMU=+0.180152D+02
 TOTAL_ENERGY:EV=-0.322679D+03
 ATOM_X_OPT:ANGSTROMS[0009]=
    0.0000    0.0000    0.0000
    0.9552    0.0000    0.0000
   -0.2536    0.9212    0.0000
 ATOM_CHARGES[0003]=
 -0.64942 +0.32469 +0.32473
 GRADIENTS:KCAL/MOL/ANGSTROM[0009]=
   0.1039  -0.0625   0.0000  -0.0706  -0.0124   0.0000  -0.0332   0.0748   0.0000
 OVERLAP_MATRIX[000021]=
 #  Lower half triangle only
   1.0000   0.0000   1.0000   0.0000   0.0000   1.0000   0.0000   0.0000   0.0000   1.0000
   0.1389   0.3753   0.0000   0.0000   1.0000   0.1388  -0.0996   0.3618   0.0000   0.2408
   1.0000
 SET_OF_MOS=       1       6
 EIGENVECTORS[000036]=
   0.8516   0.1412   0.1852   0.0000   0.3321   0.3319  -0.0001   0.6536  -0.4981   0.0000
   0.4029  -0.4030  -0.4258   0.5029   0.6600  -0.0000   0.2551   0.2551  -0.0000   0.0000
   0.0000   1.0000  -0.0000   0.0000  -0.3056  -0.3071  -0.4036   0.0000   0.5695   0.5702
  -0.0002  -0.4534   0.3451   0.0000   0.5814  -0.5808
 TOTAL_DENSITY_MATRIX[000021]=
 #  Lower half triangle only
   1.8133  -0.1879   1.4002  -0.2465   0.0651   1.4360  -0.0000  -0.0000   0.0000   2.0000
   0.3483   0.8770   0.0584  -0.0000   0.6753   0.3482  -0.1765   0.8611  -0.0000   0.0259
   0.6753
 BOND_ORDERS[000006]=
 #  Lower half triangle only
   1.7878   0.8939   0.8946   0.8939   0.0007   0.8946
 M.O.SYMMETRY_LABELS[0006]=
   1a1     1b2     2a1     1b1     3a1     2b2  
 EIGENVALUES[0006]=
  -29.615  -17.840  -14.216  -12.116    4.049    6.351
 MOLECULAR_ORBITAL_OCCUPANCIES[00006]=
 2.0000 2.0000 2.0000 2.0000 0.0000 0.0000
 ####################################
 #                                  #
 #      Normal mode analysis        #
 #                                  #
 ####################################
 ORIENTATION_ATOM_X:ANGSTROMS[0009]=
   -0.0393   -0.0515    0.0000
    0.9159   -0.0515    0.0000
   -0.2928    0.8696    0.0000
 ROTAT_CONSTS:CM(-1)[3]=    28.091241    14.483519     9.556367
 PRI_MOM_OF_I:10**(-40)*GRAM-CM**2[3]     0.996495     1.932733     2.929228
 ZERO_POINT_ENERGY:KCAL/MOL=    10.094843
 VIB._FREQ:CM(-1)[0009]=
  1395.41  2810.08  2856.07    -1.57    -1.21    -0.00   -49.32   -49.08    51.62
 NORMAL_MODES[000081]=
  -0.1818  -0.2384   0.0011   0.1128   0.6652  -0.0013   0.6114   0.2847  -0.0030  -0.2161
   0.1647   0.0007   0.6807  -0.0003  -0.0030   0.1803  -0.6560   0.0000  -0.0898  -0.1181
   0.0005   0.6830  -0.1490  -0.0028  -0.3250   0.6194   0.0007   0.9424  -0.0000  -0.0000
   0.2365   0.0000   0.0000   0.2365  -0.0000  -0.0000   0.0000   0.9424   0.0000   0.0000
   0.2365  -0.0000   0.0000   0.2365   0.0000  -0.0000   0.0000   0.9424  -0.0000  -0.0000
   0.2365   0.0000   0.0000   0.2365  -0.0015  -0.0004  -0.3345   0.0029   0.0008   0.6666
   0.0029   0.0008   0.6661  -0.0000  -0.0000  -0.0001  -0.0031  -0.0008  -0.7068   0.0031
   0.0008   0.7073   0.1552  -0.1182  -0.0005   0.0390   0.6923  -0.0010  -0.6573  -0.2213
   0.0031
 HESSIAN_MATRIX:MILLIDYNES/ANGSTROM[000045]=
 #  Lower half triangle only
   0.2941  -0.0649   0.2584   0.0000   0.0000  -0.0002  -1.0030   0.1035  -0.0000   4.4122
  -0.0737  -0.0976  -0.0000  -0.4049   0.6153  -0.0000  -0.0000   0.0003   0.0000   0.0000
  -0.0014  -0.1686   0.1553   0.0000  -0.4161   0.6986   0.0000   1.0880   0.3325  -0.9320
  -0.0000  -0.0072  -0.2263  -0.0000  -1.3173   3.9394   0.0000  -0.0000   0.0003  -0.0000
   0.0000   0.0001  -0.0000   0.0000  -0.0014
 NORMAL_MODE_SYMMETRY_LABELS[0009]=
   1A1     1B2     2A1     4????   5????   1B1     2B1     1A2     2B2  
 VIB._T_DIP:ELECTRONS[0009]=
    0.529    1.262    0.391    0.001    0.001    0.000    1.458    0.001    0.819
 VIB._TRAVEL:ANGSTROMS[0009]= 
    0.169    0.126    0.143    0.009    0.008    0.018    0.873    0.000    0.130
 VIB._RED_MASS:AMU[0009]= 
    0.547    0.519    0.490   12.626   12.626   12.626    0.598    0.504    0.489
 VIB._EFF_MASS:AMU[0009]= 
    1.696    1.501    1.149 9999.999 9999.999 9999.999    1.793    1.008   76.067
 THERMODYNAMIC_PROPERTIES_TEMPS:K[0000]= 

 ENTHALPY_TOT:CAL/MOL[0000]= 

 HEAT_CAPACITY_TOT:CAL/K/MOL[0000]= 

 ENTROPY_TOT:CAL/K/MOL[0000]= 

 H_O_F(T):KCAL/MOL[0000]= 

 ATOM_X_FORCE:ANGSTROMS[0009]=
   -0.0393   -0.0515    0.0000
    0.9159   -0.0515    0.0000
   -0.2928    0.8696    0.0000
 INT_FORCE_CONSTS:MILLIDYNES/ANGSTROM[0009]= 
    4.7049    4.1345   -0.0027
    4.4470    0.6202   -0.0014
    1.0966    3.9706   -0.0014
 CPU_TIME:SECONDS[1]=        0.02
 END OF MOPAC FILE
