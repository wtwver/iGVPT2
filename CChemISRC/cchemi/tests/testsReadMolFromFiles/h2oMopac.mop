PM7 FORCE    CHARGE=0 Singlet  BONDS AUX

Mopac file 
O        0.000000000     0.295584604     0.000036260
H        0.000000000     0.874498056     0.759794593
H       -0.000000000     0.874613161    -0.759939063



