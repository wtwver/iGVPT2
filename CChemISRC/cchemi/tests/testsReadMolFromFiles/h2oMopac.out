 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 162    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Tue Dec 15 14:34:33 2015  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  FORCE      - FORCE CALCULATION SPECIFIED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  AUX        - OUTPUT AUXILIARY INFORMATION
 *  BONDS      - FINAL BOND-ORDER MATRIX TO BE PRINTED
 *******************************************************************************
PM7 FORCE    CHARGE=0 Singlet  BONDS AUX

 Mopac file
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          0.00000000      0.29558460  *   0.00003626  *
     2       H          0.00000000  *   0.87449806      0.75979459  *
     3       H          0.00000000  *   0.87461316  *  -0.75993906  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O        0.0000    0.2956    0.0000
     2         H        0.0000    0.8745    0.7598
     3         H        0.0000    0.8746   -0.7599

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4



          HEAT OF FORMATION =     -57.799666 KCALS/MOLE


          INTERNAL COORDINATE DERIVATIVES

   NUMBER  ATOM       BOND         ANGLE      DIHEDRAL

     1       O      0.103866    -0.062450     0.000000
     2       H     -0.070631    -0.012376     0.000000
     3       H     -0.033235     0.074826     0.000000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     1.094     1.437     0.000     1.806
 HYBRID         0.196     0.257     0.000     0.323
 SUM            1.290     1.694     0.000     2.129


          GRADIENT NORM =   0.16289

          MOLECULAR WEIGHT =   18.02



          ROTATIONAL CONSTANTS IN CM(-1)

          A =   28.09124085   B =   14.48351946   C =    9.55636698



           PRINCIPAL MOMENTS OF INERTIA IN UNITS OF 10**(-40)*GRAM-CM**2

          A =        0.9965   B =        1.9327   C =        2.9292


          ORIENTATION OF MOLECULE IN FORCE CALCULATION

    NO.       ATOM         X         Y         Z

     1          O      -0.0393   -0.0515    0.0000
     2          H       0.9159   -0.0515    0.0000
     3          H      -0.2928    0.8696    0.0000


    FIRST DERIVATIVES WILL BE USED IN THE CALCULATION OF SECOND DERIVATIVES

    FOR POINT-GROUP C2v  THERE ARE   4 UNIQUE SYMMETRY FUNCTIONS.

 STEP:   1 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   2 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   3 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   4 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   5 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   6 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.99
 STEP:   7                       INTEGRAL =      0.01 TIME LEFT: 172799.99
 STEP:   8                       INTEGRAL =      0.01 TIME LEFT: 172799.99
 STEP:   9                       INTEGRAL =      0.01 TIME LEFT: 172799.99

(To convert to Hartree/Bohr^2, multiply by (10^(-21) x N x a0^2)/(4.184 x 627.51) =  0.0642)


           FORCE MATRIX IN MILLIDYNES/ANGSTROM


                  O  1       H  2       H  3
 ------------------------------------------
     O    1   6.433176
     H    2   4.079623   4.528135
     H    3   4.077971   0.850748   4.528135


          HEAT OF FORMATION =     -57.799666 KCALS/MOLE

          ZERO POINT ENERGY      10.095 KCAL/MOL


           NORMAL COORDINATE ANALYSIS (Total motion = 1 Angstrom)


   Root No.       1         2         3

                 1 A1      1 B2      2 A1  

                1395.4    2810.0    2856.0
  
           1   -0.0320   -0.0379   -0.0157
           2   -0.0420    0.0289   -0.0206
           3    0.0002    0.0001    0.0001
           4    0.0792    0.4763    0.4757
           5    0.4670   -0.0002   -0.1038
           6   -0.0009   -0.0021   -0.0020
           7    0.4292    0.1261   -0.2264
           8    0.1999   -0.4590    0.4314
           9   -0.0021    0.0000    0.0005


           MASS-WEIGHTED COORDINATE ANALYSIS (NORMAL COORDINATES)


   Root No.       1         2         3

                 1 A1      1 B2      2 A1  

                1395.4    2810.1    2856.1
  
           1   -0.1818   -0.2161   -0.0898
           2   -0.2384    0.1647   -0.1181
           3    0.0011    0.0007    0.0005
           4    0.1128    0.6807    0.6830
           5    0.6652   -0.0003   -0.1490
           6   -0.0013   -0.0030   -0.0028
           7    0.6114    0.1803   -0.3250
           8    0.2847   -0.6560    0.6194
           9   -0.0030    0.0000    0.0007


          DESCRIPTION OF VIBRATIONS


 VIBRATION    1    1A1      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1395.41        O  1 --  H  2          +50.0% ( 56.8%)     9.6%
 T-DIPOLE    0.5295        O  1 --  H  3          +50.0% ( 56.8%)     9.6%
 TRAVEL      0.1688
 RED. MASS   0.5467
 EFF. MASS   1.6955

 VIBRATION    2    1B2      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      2810.08        O  1 --  H  2          +50.0% ( 71.6%)    96.7%
 T-DIPOLE    1.2618        O  1 --  H  3          +50.0% ( 71.6%)    96.7%
 TRAVEL      0.1264
 RED. MASS   0.5195
 EFF. MASS   1.5010

 VIBRATION    3    2A1      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      2856.07        O  1 --  H  3          +50.0% ( 68.6%)    99.8%
 T-DIPOLE    0.3914        O  1 --  H  2          +50.0% ( 68.6%)    99.8%
 TRAVEL      0.1434
 RED. MASS   0.4898
 EFF. MASS   1.1489

           FORCE CONSTANT IN CARTESIAN COORDINATES (Millidynes/A)

    ATOM   CHEMICAL       X               Y               Z
   NUMBER   SYMBOL  FORCE CONSTANT  FORCE CONSTANT  FORCE CONSTANT

      1       O        4.704940        4.134546       -0.002650
      2       H        4.447030        0.620170       -0.001403
      3       H        1.096631        3.970569       -0.001403



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
